src/AppInstallerCLITests/WorkFlow.cpp

```diff
@@ -1190,7 +1191,7 @@ TEST_CASE("InstallFlow_Zip_BadRelativePath", "[InstallFlow][workflow]")
 
     // Verify Installer was not called
     REQUIRE(!std::filesystem::exists(installResultPath.GetPath()));
-    REQUIRE(installOutput.str().find(Resource::LocString(Resource::String::NestedInstallerNotFound).get()) != std::string::npos);
+    REQUIRE(installOutput.str().find(Resource::LocString(Resource::String::NestedInstallerNotFound(""_liv)).get()) != std::string::npos);

```

Comment:
Why `""_liv`?

Reply:
Updated to use the expected relative path for more accurate string loc comparison.